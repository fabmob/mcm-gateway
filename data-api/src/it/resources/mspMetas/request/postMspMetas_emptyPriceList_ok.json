{
  "hasParking": false,
  "hasStationStatus": true,
  "primaryColor": "#0054BB",
  "type": "TROTTINETTE",
  "operator": "Dott",
  "url": "https:\/\/www.cityscoot.eu\/paris\/",
  "logoUrl": "@MAAS_API_DIRECTORY_ICON@\/icon_cityscoot@3x.png",
  "priceList": null,
  "hasStation": false,
  "hasPrefParkingZone": false,
  "hasVehicle": null,
  "urlWebview": false,
  "isEnabled": null,
  "name": "Dott",
  "hasSpeedLimitZone": false,
  "logoFormat": "PNG",
  "hasNoParkingZone": false,
  "hasHold": false,
  "hasOperatingZone": false,
  "secondaryColor": "#0054BB"
}