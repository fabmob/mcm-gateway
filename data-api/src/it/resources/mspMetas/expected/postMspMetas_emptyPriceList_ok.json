{
  "hasParking": false,
  "hasStationStatus": true,
  "primaryColor": "#0054BB",
  "type": "TROTTINETTE",
  "operator": "Dott",
  "url": "https:\/\/www.cityscoot.eu\/paris\/",
  "logoUrl": "@MAAS_API_DIRECTORY_ICON@\/icon_cityscoot@3x.png",
  "priceList": null,
  "mspId": "f5e6cc23-740c-44f1-87bb-96762b35490d",
  "hasVehicle": null,
  "hasStation": false,
  "hasPrefParkingZone": false,
  "urlWebview": false,
  "isEnabled": null,
  "name": "Dott",
  "hasSpeedLimitZone": false,
  "logoFormat": "PNG",
  "hasNoParkingZone": false,
  "hasHold": false,
  "hasOperatingZone": false,
  "secondaryColor": "#0054BB"
}